dev_guide/builds/triggering_builds.adoc

```diff
@@ -32,11 +32,18 @@ link:https://docs.gitlab.com/ce/user/project/integrations/webhooks.html[GitLab],
 link:https://confluence.atlassian.com/bitbucket/manage-webhooks-735643732.html[Bitbucket],
 or Generic webhooks.
 
+{product-title} webhooks currently only support their analogous versions of the push event for each of the Git based source code management
+systems (SCMs).  All other event types are ignored.
+
+When the push events are processed, a confirmation is made as to whether the branch reference inside the event matches the branch reference in the corresponding
```

Comment:
@bfallonf - yes, the confirmation is made within openshift, specifically the openshift master.

thanks